### ISSUE TITLE
CORE: Components missing from copy function

### ISSUE THREAD

**Issue by lbross** [USFS-PNW/Fia-Biosum-Manager#86]
From @sloreno:
When you copy a core analysis scenario, a few things don’t get transferred: processor scenario selection, wood processing site selections, and treatment intensity

**Comment by lbross** [comment]
In addition to processor scenario selection and wood processing site selections, none of the information in the FVS Variables tab gets copied: Effective, Optimization, or Tie Breaker

**Comment by lbross** [comment]
I did some testing today with copying. Here are the results. Note that you must save a scenario after performing the copy, either explicitly by using the 'save' button or implicitly by running the analysis. In my tests, the wood processing site selections WERE copied. If I copied the scenario, did not save it, and re-opened it, the copied information was lost including the wood processing site selections. My guess is that this is what happened to Sara.

The processor scenario selection is not transferred. This needs to be fixed.

By default with new scenarios, treatment intensity is required but not defined. I will change this so that treatment intensity is not required by default.

It appears that even if treatment intensity is not required, the validation routine still tries to validate it. This needs to be fixed.

It seems that when copying between scenarios, if a tiebreaker is not selected then its configuration is not transferred. Since we are making an exact copy, I think this should be fixed. Even if the tiebreaker isn't required, its configuration should still be copied in case the user wants to enable it later.
